docs/admin/kubeadm.md

```diff
@@ -460,7 +460,7 @@ started.
 Once the cluster is up, you can grab the admin credentials from the master node
 at `/etc/kubernetes/admin.conf` and use that to talk to the cluster.
 
-## Use Kubeadm with other CRI runtimes
+## [Use Kubeadm with other CRI runtimes](#use-kubeadm-with-other-cri-runtimes)
```

Comment:
Jekyll already provides href targets for headings, I recommend using those directly (I think you got this one right). Adding a link to the heading actually adds a link and causes it to render strangely (see screenshot below)


If you _really_ need to add a custom href tag, please use:

```
<a id="custom-tag">
## My Heading
``` 

Your PR as it stands right now causes the heading to look different than the others:

![image](https://user-images.githubusercontent.com/159209/28593672-09417922-7142-11e7-8726-b917bb863ec3.png)